tests/memory_io.rs

```diff
@@ -0,0 +1,72 @@
+extern crate openexr;
+
+use std::io::Cursor;
+
+use openexr::{FrameBuffer, Header, ScanlineOutputFile, InputFile, PixelType};
+
+#[test]
+fn memory_io() {
+    // Target memory for writing
+    let mut in_memory_buffer = Cursor::new(Vec::<u8>::new());
+
+    // Write file to memory
+    {
+        let mut pixel_data = vec![(0.82f32, 1.78f32, 0.21f32); 256 * 256];
+
+        let mut exr_file =
+            ScanlineOutputFile::from_writer(&mut in_memory_buffer,
+                                            &Header::new()
+                                                 .set_resolution(256, 256)
+                                                 .add_channel("R", PixelType::FLOAT)
+                                                 .add_channel("G", PixelType::FLOAT)
+                                                 .add_channel("B", PixelType::FLOAT))
+                    .unwrap();
+
+        let mut fb = {
+            // Create the frame buffer
+            let mut fb = FrameBuffer::new(256, 256);
+            fb.insert_pixels(&[("R", 0.0), ("G", 0.0), ("B", 0.0)], &mut pixel_data);
+            fb
+        };
+
+        exr_file.write_pixels(&mut fb).unwrap();
+    }
+
+    // Read file from memory, and verify its contents
+    {
+        let mut pixel_data = vec![(0.0f32, 0.0f32, 0.0f32); 256 * 256];
```

Comment:
I'm not sure I understand.  It's already zero initialized I thought?  So I'm assuming you mean something different from what I'm understanding zero initialized to mean.